oidc_client/utils/utils.py

```diff
@@ -0,0 +1,133 @@
+"""General Utility Functions."""
+
+from uuid import uuid4
+
+import aiohttp
+
+from aiohttp import web
+
+from ..config import CONFIG
+from .logging import LOG
+
+
+def ssl_context():
+    """Handle application security."""
+    return None
+
+
+async def generate_state():
+    """Generate a state for authentication request and return the value for use."""
+    LOG.debug('Generate a new state for authentication request.')
+    return str(uuid4())
+
+
+async def get_from_cookies(request, key):
+    """Get a desired value from cookies."""
+    LOG.debug(f'Retrieve value for {key} from cookies.')
+
+    try:
```

Comment:
this `try` here is somewhat confusing when is `500` raised? What needs to happen for that?

Reply:
Unexpected issues with http request